src/PlatformEngines.jl

```diff
@@ -19,7 +19,7 @@ gen_download_cmd = (url::AbstractString, out_path::AbstractString) ->
     error("Call `probe_platform_engines()` before `gen_download_cmd()`")
 
 """
-    gen_unpack_cmd(tarball_path::AbstractString, out_path::AbstractString; excludelist::Union{AbstractString, Cmd} = ``)
+    gen_unpack_cmd(tarball_path::AbstractString, out_path::AbstractString; excludelist::AbstractString = nothing)
```

Comment:
`nothing` is not an `AbstractString`, so this won't work; you need to have `excludelist` be a `Union{AbstractString, Nothing}`.